src/proxy/topic/TopicProxy.ts

```diff
@@ -0,0 +1,112 @@
+import {ITopic} from '../ITopic';
```

Comment:
You need to add copyright here. Also 

`/** @ignore *//** */` under the copyright. See other proxy class files. Also you need to add `/** @internal */` to all exported classes.

Reply:
I see. Done.